modules/integration_azure-event-hub/common-modules.tf

```diff
@@ -0,0 +1 @@
+../../common/module/modules.tf
```

Comment:
The common module use `env` and `sfx_monitored` filter but we want `azure_tag_env` and `azure_tag_sfx_monitored`.